acme/localStore.go

```diff
@@ -53,45 +34,73 @@ func (s *LocalStore) Load() (cluster.Object, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err := json.Unmarshal(file, &account); err != nil {
-		return nil, err
+	if len(file) > 0 {
```

Comment:
Is it useful doing this after ` os.Open` ?

Reply:
I added this test here in the way to avoid error when `acme.json` file is empty, otherwise `acme.jason` hasd to been initialized with `{}`.